### DISCUSSION TITLE
Latest client for Win2003

### DISCUSSION THREAD

**Discussion by Zoltan74** [mickem/nscp#803] [Q&A]
Hello guys,

Can somebody enlighten me which one is the very latest NSCP for Win2003, both x32 and x64? I tried to dug it out myself one by one from the docs (where exist), but no luck. At least I understand that ver. 0.5.x works from Win2012...
Thanks a lot!

Cheers,
Zoltan

**Comment by Zoltan74** [comment]
Update: The reason for new package is the error I receive after Centreon upgrade to v.20.10.6 regarding NRPE (usually "Invalid packet version received from the server")

**Comment by rsudre** [comment]
nsclient++ web interface accepts the password, but does not leave the login screen. Could anyone help me how to solve it?

nsclient version nscp-00.5.41-x64 installed on windows 2008server.

Thanks!

**Comment by mickem** [comment]
Latest versions should support "Windows Xp" (likely some service pack) and above, also supports NRPE version 4 (if that was the other issue mentioned above).
